style.css

```diff
@@ -94,6 +95,27 @@ form label {
   height: 100%;
 }
 
+.favourite-icon-container {
+  position: absolute;
+  top: 3rem;
```

Comment:
The `top` setting is a little misleading because the form element has padding on it from where it was used further up, so the icon gets pushed down more than you'd think from just reading this.